marc_to_solr/lib/traject_config.rb

```diff
@@ -1352,21 +1352,11 @@
   fields = []
   dates = []
   embargoes = []
-  # dont check for scsb
-  MarcExtractor.cached('951knx').collect_matching_lines(record) do |field, _spec, _extractor|
-    next unless alma_951(record)
-    fields << field
-  end
-
-  MarcExtractor.cached('953abc').collect_matching_lines(record) do |field, _spec, _extractor|
-    next unless alma_953(record)
-    dates << field
-  end
-
-  MarcExtractor.cached('954ac').collect_matching_lines(record) do |field, _spec, _extractor|
-    next unless alma_954(record)
-    embargoes << field
-  end
+  # Don't index if 951$a is 'Not Available' - inactive portfolio
+  # Don't check for scsb
```

Comment:
@mzelesky I also have a question for the dont check for scsb rule. It was not added. Do I want to add it?

Reply:
@mzelesky said it's ok not to add it.